index.js

```diff
@@ -183,14 +183,20 @@ Fileupload.prototype.handle = function (ctx, next) {
         return req.resume();
     } else if (req.method === "GET") {
 
-        if (this.events.get) {
-            this.events.get.run(ctx, domain, function(err) {
-                if (err) return ctx.done(err);
-                self.get(ctx, next);
-            });
-        } else {
-            this.get(ctx, next);
-        }
+		this.get(ctx, function(err, result) {
```

Comment:
Nice, you also fixed the issue that the event is called before the actual get request. Was is on purpose?